jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,147 @@
+---
+
+- job:
+    name: osd-testing-nightly-trigger
+    project-type: pipeline
+    description: "Nightly pipeline trigger for QE OSD cluster (rhmi-qe1) testing."
+    sandbox: false
+    concurrent: false
+    triggers:
+      - timed: "H(0-10) 6 * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+      - inject:
+          properties-content: |
+            CLUSTER_NAME=rhmi-qe1
+            RECIPIENTS=integreatly-qe@redhat.com
+            INSTALLATION_REPOSITORY=https://github.com/integr8ly/installation.git
+            INSTALLATION_BRANCH=master
+            ROUTER_SHARD=f2d1
+            OPENSHIFT_TOKEN=
```

Comment:
is there any point in providing empty value here?

Reply:
Yes, for the time being this will the same as with PDS, we need to update values in PDS nightly triggers too.